Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_golang_1.22 AS builder
```

Comment:
Should this file be named `Dockerfile.openshift`? Since the repository didn't have a Dockerfile, I used the same name.